### PR TITLE
Add Testery set up to Tutorials

### PR DESCRIPTION
**Pull request by dacharyc**
Alrighty, @q0rban , here's an initial pass at adding a "Testery.io Integration" doc to the docs site. Can you take a look at the example `config.yml` and make any corrections you spot, and/or comments/fixes for the overall page?

### CODE REVIEW

File: content/starter-configs/tutorials/testery.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        - wp --allow-root --path="${DOCROOT}" config create --dbhost=mysql --dbname=tugboat --dbuser=tugboat
+          --dbpass=tugboat --force
+
+        # Update permalinks to remove the index.php.
+        - wp --allow-root --path="${DOCROOT}" option set permalink_structure /%postname%/
+        - wp --allow-root --path="${DOCROOT}" rewrite flush --hard
+
+        # Install pip so you can get the Testery CLI
+        - apt-get install python3
+        - apt-get install python3-pip
```
Comment (q0rban) [inline content/starter-configs/tutorials/testery.md:96]: This should be:

```
- apt-get install -y python3
- apt-get install -y python3-pip
```

OR

```
- apt-get install -y python3 python3-pip
```

File: content/starter-configs/tutorials/testery.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        # Link the docroot
+        - ln -snf "${TUGBOAT_ROOT}/docroot" "${DOCROOT}"
+
+        # Configure wordpress to work with tugboat
+        - wp --allow-root --path="${DOCROOT}" config create --dbhost=mysql --dbname=tugboat --dbuser=tugboat
+          --dbpass=tugboat --force
+
+        # Update permalinks to remove the index.php.
+        - wp --allow-root --path="${DOCROOT}" option set permalink_structure /%postname%/
+        - wp --allow-root --path="${DOCROOT}" rewrite flush --hard
```
Comment (q0rban) [inline content/starter-configs/tutorials/testery.md:92]: The applicable config gets lost in here. What do you think about removing all of the detail above and doing something like this instead:

```yaml
services:
  php:
    commands:
      init:
        ...
        # Install pip so you can get the Testery CLI.
        - apt-get update
        - apt-get install -y python3 python3-pip
      build:
        ...
        # Check for upgrades to the Testery command line interface
        - pip3 install testery --upgrade
        # Set up Testery to run tests against the Tugboat environment, using Tugboat environment variables
        - testery update-environment --create-if-not-exists --token "$TESTERY_TOKEN" --key "${TUGBOAT_PREVIEW}" --name
          "${TUGBOAT_PREVIEW}" --variable "TUGBOAT_DEFAULT_SERVICE_URL=${TUGBOAT_DEFAULT_SERVICE_URL}"
        # Start a test run.
        - testery create-test-run --token "$TESTERY_TOKEN" --git-ref "$TUGBOAT_PREVIEW_SHA" --project "testery"
          --environment "${TUGBOAT_PREVIEW}"
```

File: content/starter-configs/tutorials/testery.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+- [Name your Service](/setting-up-services/how-to-set-up-services/name-your-service/)
+- [Specify a Service image](/setting-up-services/how-to-set-up-services/specify-a-service-image/)
+- [Leverage Service commands](/setting-up-services/how-to-set-up-services/leverage-service-commands/)
+- [Define a default Service](/setting-up-services/how-to-set-up-services/define-a-default-service/)
+- [Set the document root path](/setting-up-services/how-to-set-up-services/set-the-document-root-path/)
+- [Set up remote SSH access](/setting-up-tugboat/select-repo-settings/#set-up-remote-ssh-access)
+- [Preview build process phases (`init`, `update`, `build`)](/building-a-preview/preview-deep-dive/how-previews-work/#the-build-process-explained)
+- [How Base Previews work](/building-a-preview/preview-deep-dive/how-previews-work/#how-base-previews-work)
+
+## Start Building Previews!
```
Comment (q0rban) [inline content/starter-configs/tutorials/testery.md:166]: Maybe we should say "Start testing your Previews!" or something? Or just remove this paragraph?

**Comment by dacharyc** [comment]
Alrighty, I've updated the Testery doc, but I've also removed the deprecated testing code snippets we discussed yesterday in the Slack, @q0rban . I've added permanent redirects for those to the Testery page; can you verify I've done those rewrite rules correctly, @bchavet ?

File: static/.htaccess [hunk 4]
```diff
@@ -92,5 +92,7 @@
     RewriteRule ^starter-configs/code-snippets/static-hugo/?$ %{ENV:CWD}starter-configs/examples/static-hugo/ [L,R=301]
     RewriteRule ^starter-configs/code-snippets/generic-lamp/?$ %{ENV:CWD}starter-configs/examples/generic-lamp/ [L,R=301]
     RewriteRule ^starter-configs/code-snippets/static-jekyll/?$ %{ENV:CWD}starter-configs/examples/static-jekyll/ [L,R=301]
+    RewriteRule ^starter-configs/code-snippets/functional-tests/?$ %{ENV:CWD}starter-configs/tutorials/testery/ [L,R=301]
+    RewriteRule ^starter-configs/code-snippets/simpletest/?$ %{ENV:CWD}starter-configs/tutorials/testery/ [L,R=301]
```
Comment (q0rban) [inline static/.htaccess:96]: These work great. Tested on Tugboat.

**Pull request merged**